src/main/resources/application.yml

```diff
@@ -102,6 +111,8 @@ INFOTRYGD_REPLIKA_API_URL: https://infotrygd-enslig-forsoerger.nais.adeo.no
 
 FRONTEND_OPPGAVE_URL: https://ensligmorellerfar.prod-fss.nais.io/fagsak
 
+GYLDIG_SERVICE_BRUKER: srvArena
```

Comment:
Kan dette bli en liste etterhvert? Isåfall; `GYLDIGE_SERVICE_BRUKERE` er kanskje et mer egnet navn ?

Reply:
👍 Ja denne håndterer liste nå, så endrer til GYLDIGE_SERVICE_BRUKERE